cds_migrator_kit/videos/weblecture_migration/transform/transform_files.py

```diff
@@ -99,127 +94,21 @@ def _add_files_to_file_json(self, json_key, folder, files_list):
-                if {keyword, pair} == found_keywords:
-                    self.presenter_str = keyword
-                    self.presentation_str = pair
-                    self.use_composite = True
-                    return True
-        
-        return False
-    
-    def _find_master_data_folder_path(self, path):
-        """Find the full path of the master_path(comes from marcxml)"""        
-        search_paths = [
-            self.master_folder / path, # master_folder/year/id
-            self.master_folder / path / "media" # master_folder/year/id/media
-        ]
-
-        try:
-            # Try to find video files in folder
-            for search_path in search_paths:
-                all_files = [f.name for f in Path(search_path).iterdir() if f.is_file() and f.suffix == ".mp4"]
-                if all_files:
-                    return search_path # Files are found
-        except FileNotFoundError:
-            # No files are found, raise an error
-            raise ManualImportRequired(
-                message="Master folder couldn't be found!",
-                stage="transform",
-                field="master_data",
-                recid=self.recid,
-                value=search_path,
-                priority="critical",
-            )
-    
-    def _check_master_data_folder(self):
-        """Check the master_data folder and set the files.
-        
-        - If presenter and presentation found, add them to additional files.
-        - If presenter and presentation not found tries to get the `video.mp4` file as master.
-        """
-        # Get all files in the master data folder
-        all_files = self._get_all_files_in_folder(self.master_data_folder)
-
-        # Presenter and presentation files are found add all files as additional
-        # Composite will be used as main file
-        if self._check_presenter_presentation_exists(all_files):
-            # Add all files as additional
-            self._add_files_to_file_json(
-                json_key="additional_files", 
-                folder=self.master_data_folder, 
-                files_list=all_files
-            )
-            return
-        
-        # Presenter and presentation doesn't exists, try to find the file named "video.mp4" or "presenter.mp4"
-        video_files = [file for file in all_files if file.lower() == "video.mp4" or file.lower() == "presenter.mp4"]
-        if len(video_files) == 1:
-            self.transformed_files_json["master_video"] = str(self.master_data_folder / video_files[0])
-        else:
-            raise ManualImportRequired(
-                message="Master file couldn't be found!",
-                stage="transform",
-                recid=self.recid,
-                value="master_video",
-                priority="critical",
-            )
-        
+            if self.composite_str in file_name:
+                self.use_composite = True
+                return 
```

Comment:
better to 
```suggestion
                return True
```
than mutating the `self.use_composite`. This is the only place that you are setting the `self.use_composite`?